src/ml_performance_monitoring/monitor.py

```diff
@@ -75,6 +76,8 @@ def __init__(
             )
         if not isinstance(model_name, str):
             raise TypeError("model_name instance type must be str")
+        if not isinstance(model_version, str):
+            raise TypeError("model_version instance type must be str")
```

Comment:
@YehezkelShay perhaps we can also validate that `model_version` is not an empty string

Reply:
Thanks for the comment,  good idea.